### PR TITLE
Hide sidebar on demand

### PR DESCRIPTION
**Pull request by fonglh**
Add hide/show sidebar chevrons.

hideshowsidebar

### CODE REVIEW

**Comment by fonglh** [comment]
Possible enhancement would be to use custom animations so the sidebar slides left and right instead.

See https://bootstrapious.com/p/bootstrap-sidebar

File: app/views/course/courses/show.html.slim [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= div_for(current_course) do
+= div_for(current_course, class: 'page-header') do
```
Comment (jeremyyap) [inline app/views/course/courses/show.html.slim]: What is the purpose of this `page-header` class?

**Comment by jeremyyap** [comment]
Code looks fine but the effect is a bit ugly. Especially the chevrons' animations which look jumpy/glitchy

File: app/views/course/courses/show.html.slim (hunk 1 above)
Comment (fonglh) [reply app/views/course/courses/show.html.slim]: Pushes the course title a bit lower down so it is not blocked by the chevron when the sidebar is hidden.

**Pull request merged**